quasar/src/components/chat/QChatMessage.js

```diff
@@ -1,6 +1,6 @@
 import Vue from 'vue'
 
-import slot from '../../utils/slot.js'
+// import slot from '../../utils/slot.js'
```

Comment:
why?

Reply:
because slot is declared but never used, i know it also is out of scope of that PR.

Reply:
is this outdated? cause there's no such import in QChatMessage.